src/js/views/patients/patient/sidebar/patient-sidebar.scss

```diff
@@ -1,5 +1,10 @@
 .patient-sidebar {
   padding: 40px 24px 0;
+  width: 100%;
```

Comment:
is this needed? isn't `.patient-sidebar` a `div`?

Reply:
It is a div, but it's inside a flex div which caused it to shrink to the width of the text inside it. Visually, it's not noticeable. But once the spinner was added it was off center. I'll remove this now that we don't need the spinner.